ui/app/styles/components/status-report-indicator.scss

```diff
@@ -0,0 +1,86 @@
+.status-report-indicator {
+  display: inline-flex;
+  align-items: center;
+  cursor: default;
+
+  &:focus {
+    box-shadow: 0 0 0 3px var(--pds-focus-color);
+    outline: none;
+  }
+
+  &__badge {
+    width: scale.$sm--2;
+    height: scale.$sm--2;
+    border-radius: 50%;
+    background: color.$black;
+    margin-right: scale.$sm--3;
+  }
+
+  &__label {
+    @include Typography.Interface(S);
+  }
+
+  // Status: Unknown
+
+  &--unknown &__badge {
+    background: color.$ui-gray-300;
+  }
+  &--unknown &__label {
+    color: rgb(var(--text-muted));
+  }
+
+  // Status: Alive
+
+  &--alive &__badge {
+    background: color.$green-500;
+    animation: pulse 2s infinite;
+    @media (prefers-reduced-motion: reduce) {
+      animation: none;
+    }
+  }
+  &--alive &__label {
+    color: color.$green-700;
+    @media (prefers-color-scheme: dark) {
```

Comment:
This is great. We could simplify the colors by declaring them in variables in a `:root` pseudo-class up top, but frankly this is already fine and readable

Reply:
Thanks!

I tried to reuse as many of the `:root` variables as I could keep straight in my head. There are probably more opportunities to consolidate but I think this does the trick for now.